lib/RoleDetails/RoleDetailsCapabilitiesAccordion/RoleDetailsCapabilitiesAccordion.test.js

```diff
@@ -0,0 +1,56 @@
+import { cleanup, render } from '@folio/jest-config-stripes/testing-library/react';
+import '@folio/jest-config-stripes/testing-library/jest-dom';
+
+import renderWithRouter from '../../../test/jest/helpers/renderWithRouter';
```

Comment:
I would suggest adding aliases for such pathes into eslint config: (e.g. https://github.com/folio-org/ui-orders/blob/master/.eslintrc#L9-L19)
```
"import/resolver": {
      "alias": {
        "map": [
          ["__mock__", "./test/jest/__mock__"],
          ["fixtures", "./test/jest/fixtures"],
          ["helpers", "./test/jest/helpers"]
        ]
      }
    }
```

it will simplify unils for tests